src/utility/errors.ts

```diff
@@ -1,10 +1,13 @@
 import { AxiosError } from 'axios';
+import { isAxiosError } from './axios';
 
-export function formatMessage(error: Error, message: string, defaultErrorMessage = 'Internal Server Error') {
+export function formatMessage(error: Error, message = '', defaultErrorMessage = 'Internal Server Error') {
   let errorMessage = defaultErrorMessage;
-  const axiosError = error as AxiosError;
-  if (axiosError?.response?.data?.errors && axiosError?.response.data.errors.length > 0) {
-    errorMessage = axiosError?.response.data.errors[0].message;
+  if (isAxiosError(error)) {
+    const axiosError = error as AxiosError;
+    if (axiosError?.response?.data?.errors && axiosError?.response.data.errors.length > 0) {
+      errorMessage = axiosError?.response.data.errors[0].message ?? defaultErrorMessage;
+    }
```

Comment:
I think we could make this a little easier to read.
```suggestion
    const errors = error?.response?.data?.errors
    if (errors && errors.length > 0) {
      errorMessage = errors[0].message ?? defaultErrorMessage;
    }
```